tests/others/test_metrics.py

```diff
@@ -60,11 +58,9 @@ def test_connections_manager_integration(self):
         self.tmpdirs.append(tmpdir)
         wallet = Wallet(directory=tmpdir)
         wallet.unlock(b'teste')
-        manager = HathorManager(self.clock, tx_storage=tx_storage, wallet=wallet)
+        manager = self.create_peer('testnet', tx_storage=tx_storage, wallet=wallet)
 
-        manager.metrics.start()
```

Comment:
Why you don't need this anymore?

Reply:
We are using `self.create_peer(...)`, which runs `manager.start()`, which runs `manager.metrics.start()`.